graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -298,15 +298,29 @@ public long numberOfMessages(String indexName) throws IndexNotFoundException {
 
     public boolean exists(String indexName) {
```

Comment:
Please add a short Javadoc comment what the method is supposed to do (and what not).